change/@fluentui-react-provider-0c6fd81b-10fb-4962-aafe-32fcf9a1970f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28803&commitId=72fb285112886b7ea6e98794433fdf6954605dbf&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.customSize+image.normal.chromium.png|84| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3357| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|1340| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|52| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>